packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -14,75 +14,108 @@ import {
   cloudinaryAuthBannerBackground1440w as laptopBg,
   cloudinaryAuthBannerBackground1920w as desktopBg,
 } from '../../lib/image';
+import { checkIsBrowser, checkIsExtension, UserAgent } from '../../lib/func';
+import { featurePostBannerExtensionPrompt } from '../../lib/featureManagement';
+import { useConditionalFeature } from '../../hooks';
+import { GetExtensionButton } from '../buttons/GetExtensionButton';
 
 const Section = classed('div', 'flex flex-col');
 
 export function AuthenticationBanner({
   children,
 }: PropsWithChildren): ReactElement {
+  const isCompatibleBrowser =
+    (checkIsBrowser(UserAgent.Chrome) || checkIsBrowser(UserAgent.Edge)) &&
+    !checkIsExtension();
+
+  const { value: showExtensionCTA } = useConditionalFeature({
+    feature: featurePostBannerExtensionPrompt,
+    shouldEvaluate: isCompatibleBrowser,
+  });
+
   const { showLogin } = useAuthContext();
 
   return (
     <BottomBannerContainer
       className={classNames(
-        'gap-24 border-t border-accent-cabbage-default py-10 shadow-3 laptopL:gap-32',
```

Comment:
Are these classes removed as intended?

Reply:
@sshanzel Yeah, design has 40px gap between the sections and Tsahi specifically mentioned the gap sized in the sync 😄